### PR TITLE
Fix/event link and cell size

### PR DESCRIPTION
**Pull request by autumnfjeld**
# Description
Two small updates to improve usability.

### 1) Use tracks event link , not live link
image
The event links in  were linking to the "live" page:
https://mc.a8c.com/tracks/live/?eventname=calypso_theme_sheet_style_variation_click
instead of the actual event page:
https://mc.a8c.com/tracks/events/event.php?eventname=calypso_theme_sheet_style_variation_click

The actual event page is more useful , it has richer information and it also has a link to bothe the "live" and "trend" view.

If there was some reason for linking to the live page, let me know. :) 

### 2) Made the table more compact
I often use this when I'm just on my laptop and a more condensed view is easier to read when monitor space is minimal.

cc: @Automattic/lego

### CODE REVIEW

File: Tracks Vigilante/manifest.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "Tracks Vigilante",
     "manifest_version": 3,
-    "version": "1.7",
+    "version": "1.71",
```
Comment (taipeicoder) [inline Tracks Vigilante/manifest.json:4]: We can either bump it to `1.7.1` or `1.8` but `1.71` is a bit odd 🙂

**Review by taipeicoder — APPROVED** [review]
Tested and it works well 👍

**Review by okmttdhr — APPROVED** [review]
✅

File: Tracks Vigilante/manifest.json (hunk 1 above)
Comment (autumnfjeld) [reply Tracks Vigilante/manifest.json:4]: Ha, ya I wasn't quiet sure what increment it should be.  :)

**Pull request merged**